SmartDeviceLink/SDLSendLocation.h

```diff
@@ -49,7 +48,7 @@
  *
  * Contains String, Optional, Max Array Length = 4, Max String Length = 500
  */
-@property (copy, nonatomic) NSArray *addressLines;
+@property (copy, nonatomic) NSArray<NSString *> *addressLines;
```

Comment:
Should it be noted that OASIS address is now used to actually find the place if provided, this would be purely for decoration / display?